.github/workflows/template-cleanup.yml

```diff
@@ -16,7 +16,9 @@ jobs:
     steps:
 
       - name: Checkout
-        uses: actions/checkout@v2
+        uses: actions/checkout@v4
+        with:
+          persist-credentials: false
```

Comment:
I didn't know about this option. 🧐

Reply:
Found out about it when updating `ad-m/github-push-action`: https://github.com/ad-m/github-push-action#example-workflow-file (the PR adding those instructions explained)